demo/src/main.ts

```diff
@@ -55,13 +54,13 @@ const settingsModel = monaco.editor.createModel(
 createConfiguredEditor(document.getElementById('settings-editor')!, {
   model: settingsModel
 })
-settingsModel.onDidChangeContent(debounce(1000, () => {
-  updateUserConfiguration(settingsModel.getValue())
+settingsModel.onDidChangeContent(debounce(1000, async () => {
+  await updateUserConfiguration(settingsModel.getValue())
 }))
-updateUserConfiguration(settingsModel.getValue())
+await updateUserConfiguration(settingsModel.getValue())
```

Comment:
top level awaits are probably a bad idea here

Reply:
Ok, then void to evade the eslint errors. Line 87 too btw

Reply:
Done ⬇️